### ISSUE TITLE
Cron doesn't start at the right time after daylight saving time change

### ISSUE THREAD

**Issue by skinnycat-software** [Prospress/action-scheduler#885]
Existing jobs running at a set time daily using a cron schedule ( ie setup using as_schedule_cron_action ) do not run at the proper time after Daylight saving time change and keep being rescheduled daily at the wrong time.

All the jobs are running an hours early after 11/06/2022. When canceling the scheduled jobs, the new ones start at the correct time.

I would expect the daily rescheduling after a job run to take into account the daylight saving time change and schedule the next job accordingly.

**Comment by barryhughes** [comment]
Hi @skinnycat-software,

Yes, this is a problem (cron scheduling is based on UTC, so there is an apparent shift of one hour as sites enter and leave DST) and essentially boils down to the same thing [encapsulated in this issue](https://github.com/woocommerce/action-scheduler/issues/765) (which also contains some user-suggested workarounds you could try in the meantime).

Resolving this is going to be hard and may present some backward-compatibility challenges, and unfortunately is not something we can prioritize at this time.

**Comment by dwainm** [comment]
I'm going to close this issue as a won't fix, due to the complexity of fixing it. Our aim for cron jobs is that they eventually run as close to the desired time as possible. While there might be ways to counter DST offset, from our perspective there isn't a significant benefit  to warrant the complexity and possible edge cases that could come form ensuring exact timing that respects DST and other server configurations.

**Comment by skinnycat-software** [comment]
Fair enough. However I suggest to make it very clear in the documentation that  cron-like schedule should only be based on UTC time and will not account for DST offset.
